pkg/identityManager/fake/reader.go

```diff
@@ -51,6 +59,25 @@ func (i *IdentityReader) GetConfig(remoteCluster discoveryv1alpha1.ClusterIdenti
 	return nil, fmt.Errorf("remote cluster ID %v not found", remoteCluster.ClusterID)
 }
 
+func (i *IdentityReader) GetSecretNamespacedName(remoteCluster discoveryv1alpha1.ClusterIdentity,
+	namespace string) (types.NamespacedName, error) {
+	if ns, found := i.namespaces[remoteCluster.ClusterID]; found {
+		if secretName, found := i.secretNames[remoteCluster.ClusterID]; found {
+			return types.NamespacedName{
+				Namespace: ns,
+				Name:      secretName,
+			}, nil
+		}
+		return types.NamespacedName{}, fmt.Errorf("secret name for remote cluster ID %v not found", remoteCluster.ClusterID)
+	}
+	return types.NamespacedName{}, fmt.Errorf("remote cluster ID %v not found", remoteCluster.ClusterID)
+}
```

Comment:
@aleoli the namespace argument is still useful?

Reply:
not in the fake implementation, but here is used https://github.com/liqotech/liqo/blob/11ec09126d779033fd70686f3fe85709c0975622/pkg/identityManager/client.go#L54